eo/environment.yml

```diff
@@ -0,0 +1,6 @@
+name: birdy
+channels:
+  - conda-forge
+  - defaults
+# add any new dependencies here
+# dependencies:
```

Comment:
I was wondering what was the best course of action. To wait and get some feedback on what to add here (using a confluence page). Or to release the images first, even if the starter pack is not defined yet.

I wanted to have the images released first, so I could start working on birdhouse-deploy right away, and adapt it to use those choice of images.

Reply:
Once the Confluence page is created, maybe link add the page link to the readme of the current repo. 
I suppose this page will be used only at the beginning, since by after it will be directly via PRs that people suggest packages.. But always nice to keep some traceability in here.